stripe.go

```diff
@@ -167,9 +152,8 @@ type BackendImplementation struct {
 	Type              SupportedBackend
 	URL               string
 	HTTPClient        *http.Client
+	LeveledLogger     LeveledLoggerInterface
 	MaxNetworkRetries int
-	LogLevel          int
-	Logger            Printfer
```

Comment:
This is technically a backwards-incompatible change, but I think I'm going to release as a minor anyway. There's no plausible reason to ever access a `BackendImplementation` directly (and especially not the `LogLevel` and `Logger` fields), and I've had a note in its docblock that we're going to export it soon forever now.

Reply:
Works for me, though the cost of doing a major is quite low and so when in doubt, maybe a major is better? We could also wait for the next breaking change to release this.

But ultimately fine with your approach.